.buildkite/pipeline.build_chaos.yml

```diff
@@ -0,0 +1,30 @@
+#ci:group=:kubernetes: :mending_heart: chaos tests
+
+- label: "chaos test {{matrix.workload}} under {{matrix.fault}}"
+  conditions: ["RAY_CI_LINUX_WHEELS_AFFECTED"]
+  instance_size: medium
+  commands:
+    - |
+      cleanup() {
+        if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi
+        kind delete cluster
+      }
+      trap cleanup EXIT
+    - echo "--- Preparing chaos testing environment"
+    - ./python/ray/tests/chaos/prepare_env.sh
+    # First run: no faults
+    - echo "--- Running the workload without faults"
+    - "./python/ray/tests/chaos/{{matrix.workload}}.sh"
+    # Second run: with faults
+    - kubectl apply -f python/ray/tests/chaos/{{matrix.fault}}.yaml
```

Comment:
is it possible to configure num_cpus for these workloads?

Reply:
what do you mean by num_cpus? You mean the ray tasks' resource needs?

Reply:
It'd be great if we can configure num cpus  & num workers from the k8s cluster.